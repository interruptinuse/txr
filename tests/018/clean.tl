(trace remove-path)

(each ((*load-path* '(nil "/tmp/foo.tlo")))
  (each ((name '#"nabuchodonosor.tl nabuchodonosor.txr \
                  nabuchodonosor.tlo nabuchodonosor.tlo.gz \
                  nabuchodonosor.abc nabuchodonosor"))
    (prinl ^(let ((*load-path* , *load-path*)) (clean-file ,name)))
    (put-line "-->")
    (clean-file name)
    (put-line)))
