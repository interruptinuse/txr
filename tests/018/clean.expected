(let ((*load-path*
       ()))
  (clean-file "nabuchodonosor.tl"))
-->
(remove-path ("nabuchodonosor.tlo" nil)
  nil)
(remove-path ("nabuchodonosor.tlo.gz" nil)
  nil)

(let ((*load-path*
       ()))
  (clean-file "nabuchodonosor.txr"))
-->
(remove-path ("nabuchodonosor.tlo" nil)
  nil)
(remove-path ("nabuchodonosor.tlo.gz" nil)
  nil)

(let ((*load-path*
       ()))
  (clean-file "nabuchodonosor.tlo"))
-->
(remove-path ("nabuchodonosor.tlo" nil)
  nil)

(let ((*load-path*
       ()))
  (clean-file "nabuchodonosor.tlo.gz"))
-->
(remove-path ("nabuchodonosor.tlo.gz" nil)
  nil)

(let ((*load-path*
       ()))
  (clean-file "nabuchodonosor.abc"))
-->
(remove-path ("nabuchodonosor.abc.tlo" nil)
  nil)
(remove-path ("nabuchodonosor.abc.tlo.gz" nil)
  nil)

(let ((*load-path*
       ()))
  (clean-file "nabuchodonosor"))
-->
(remove-path ("nabuchodonosor.tlo" nil)
  nil)
(remove-path ("nabuchodonosor.tlo.gz" nil)
  nil)

(let ((*load-path*
       "/tmp/foo.tlo"))
  (clean-file "nabuchodonosor.tl"))
-->
(remove-path ("/tmp/nabuchodonosor.tlo" nil)
  nil)
(remove-path ("/tmp/nabuchodonosor.tlo.gz" nil)
  nil)

(let ((*load-path*
       "/tmp/foo.tlo"))
  (clean-file "nabuchodonosor.txr"))
-->
(remove-path ("/tmp/nabuchodonosor.tlo" nil)
  nil)
(remove-path ("/tmp/nabuchodonosor.tlo.gz" nil)
  nil)

(let ((*load-path*
       "/tmp/foo.tlo"))
  (clean-file "nabuchodonosor.tlo"))
-->
(remove-path ("/tmp/nabuchodonosor.tlo" nil)
  nil)

(let ((*load-path*
       "/tmp/foo.tlo"))
  (clean-file "nabuchodonosor.tlo.gz"))
-->
(remove-path ("/tmp/nabuchodonosor.tlo.gz" nil)
  nil)

(let ((*load-path*
       "/tmp/foo.tlo"))
  (clean-file "nabuchodonosor.abc"))
-->
(remove-path ("/tmp/nabuchodonosor.abc.tlo" nil)
  nil)
(remove-path ("/tmp/nabuchodonosor.abc.tlo.gz" nil)
  nil)

(let ((*load-path*
       "/tmp/foo.tlo"))
  (clean-file "nabuchodonosor"))
-->
(remove-path ("/tmp/nabuchodonosor.tlo" nil)
  nil)
(remove-path ("/tmp/nabuchodonosor.tlo.gz" nil)
  nil)

